src/test/java/com/blackducksoftware/integration/hub/alert/mock/entity/MockHipChatEntity.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2017 Black Duck Software Inc.
+ * http://www.blackducksoftware.com/
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Black Duck Software ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Black Duck Software.
+ */
+package com.blackducksoftware.integration.hub.alert.mock.entity;
+
+import org.json.JSONException;
+import org.junit.Test;
+
+import com.blackducksoftware.integration.hub.alert.datasource.entity.distribution.HipChatDistributionConfigEntity;
+import com.google.gson.JsonObject;
+
+public class MockHipChatEntity implements MockEntityUtil<HipChatDistributionConfigEntity> {
+    private final Integer roomId;
+    private final Boolean notify;
+    private final String color;
+    private final Long id;
+
+    public MockHipChatEntity() {
+        this(11, false, "black", 1L);
+    }
+
+    private MockHipChatEntity(final Integer roomId, final Boolean notify, final String color, final Long id) {
+        this.roomId = roomId;
+        this.notify = notify;
+        this.color = color;
+        this.id = id;
+    }
+
+    @Test
+    public void tes() throws JSONException {
```

Comment:
Can you give this test a real name?

Reply:
These tests will be gone once I change from interface to an abstract class.